recipe/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "relationalai" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ad4f1960f37f41e8220915c7daa03889f192cf8b07145a295d1e79e0300a4a6a
+  patches:
+    - patches/0001-remove-debugger-command.patch
+
+build:
+  number: 0
+  # snowflake-snowpark-python and snowflake-connector-python not on s390x
+  skip: true  # [py<310 or py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  entry_points:
+    - rai = relationalai.tools.cli:cli
+    - gentest = gentest.cli.__main__:main
+
+requirements:
+  build:
+    - patch       # [not win]
+    - m2-patch    # [win]
+  host:
+    - python
+    - hatchling
+    - pip
+  run:
+    - python
+    - snowflake-connector-python
+    - snowflake-snowpark-python
+    - rai-sdk >=0.7.5
+    - rich
+    # see patch
+    # - nicegui
+    - numpy
+    - pandas
+    - colorama
+    - inquirerpy
+    - click
+    - gravis
+    - bytecode
+    - toml
+    - tomlkit
+    - requests
+    - pyarrow
+    - websockets
+    - aiohttp
+    - packaging
+
+test:
+  imports:
+    - relationalai
+  requires:
+    - pip
+    - pytest
+    - pytest-mock
+    - hypothesis
+  source_files:
+    - tests/unit
+    - tests/conftest.py
+  commands:
+    - pip check
+    - rai --help
+    - rai version
+    # Must show "No debugger found, use the new debugger with --new"
+    - rai debugger
+    # We run only this one test, because all the other will require credentials
+    # for the snowflake/azure backend, and need to connect to their services.
+    #
+    # Skip test_new_profile because the snapshot is similar but not exactly
+    # matching the computed value (same fields, different order).
+    - pytest -vv tests/unit -k "not test_new_profile"
+
+about:
+  summary: The Python library for building and querying knowledge graphs with RelationalAI.
+  description: |
+    The Python library for building and querying knowledge graphs with RelationalAI.
+  license: Apache-2.0
```

Comment:
What is the source of the license? It's not mentioned in the source code

Reply:
Internal. See comment https://anaconda.atlassian.net/browse/PKG-4599?focusedCommentId=218479